android-timetracker/src/com/markuspage/android/atimetracker/Tasks.java

```diff
@@ -58,11 +58,19 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
+//import org.apache.commons.logging.LogFactory;
```

Comment:
This commented out code would be better to clean out. The extra blank lines as well.